instrumentation/github.com/emicklei/go-restful/otelrestful/restful.go

```diff
@@ -45,7 +45,7 @@ func OTelFilter(service string, opts ...Option) restful.FilterFunction {
 		spanName := route
 
 		opts := []oteltrace.SpanStartOption{
-			oteltrace.WithAttributes(semconvutil.HTTPServerRequest(service, r)...),
+			oteltrace.WithAttributes(semconvutil.HTTPServerRequest(service, r, semconvutil.HTTPServerRequestOptions{})...),
```

Comment:
semconvutil if for semantic conventions v1.20.0. We need to support v1.26.0 now, and semconvutil will be removed after the next release.
Please use the semconv package instead.

Reply:
Related (this PR is actually probably blocked by) https://github.com/open-telemetry/opentelemetry-go-contrib/pull/6951

Reply:
@dmathieu I'm a little confused, sorry. Are you removing the entire `semconvutil` package, including this file https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/internal/shared/semconvutil/httpconv.go.tmpl ?

And you're saying I should modify https://github.com/open-telemetry/opentelemetry-go-contrib/blob/main/internal/shared/semconv/httpconv.go.tmpl instead?

Reply:
Ok, I'll fix this.

Should I still leave the fix in `semconvutil` _in addition to_ `semconv`? Or `semconv` only?

Reply:
Done, updated both semconv and semconvutil. I understand now that semconvutil is pretty much unused, however.

Reply:
Yes, we should have the fix in both. But semconvutil will be removed after the next release.

Reply:
Ok, should be ready then

Reply:
Could you please allow the CI to run too?